templates/components/form/fields_macros.html.twig

```diff
@@ -400,15 +400,41 @@
 {% endmacro %}
 
 {% macro passwordField(name, value, label = '', options = {}) %}
-   {% set options = {'autocomplete': 'new-password'}|merge(options) %}
+   {% set options = {
+      'autocomplete': 'new-password',
+      'is_disclosable': false,
+      'rand': options.rand ?? random()
+   }|merge(options) %}
+   {% set options = {
+      'id': options.id is defined ? options.id : name ~ random(),
+      'clearable': options.clearable is defined ? options.clearable : not options.is_disclosable,
+   }|merge(options) %}
+
    {% set field %}
-      <input type="password" id="%id%"
+      <input type="password" id="{{ options.id }}"
              class="form-control"
              name="{{ name }}"
+             value="{{ options.is_disclosable ? value : '' }}"
          {{ options.readonly ? 'readonly' : '' }}
          {{ options.disabled ? 'disabled' : '' }}
          {{ options.required ? 'required' : '' }} />
    {% endset %}
+   {% set btn_group %}
+      &nbsp;<i class="far fa-eye pointer disclose" onmousedown="showField('{{ options.id }}')"
+               onmouseup="hideField('{{ options.id }}')"
+               onmouseout="hideField('{{ options.id }}')"></i>
+      &nbsp;<i class="fa fa-paste pointer disclose" onclick="copyToClipboard('{{ options.id }}')"></i>
```

Comment:
`showField`, `hideField` and `copyToClipboard` are declared in `templates/components/form/item_device.html.twig` and will not be accessible in some contexts. The should be moved in `common.js` file.